### PR TITLE
RAINCATCH-301: Protractor tests for the raincatcher-file module

### PR DESCRIPTION
**Pull request by JameelB**
# What 
Added protractor tests for File E2E

# JIRA Ticket
https://issues.jboss.org/browse/RAINCATCH-301

### CODE REVIEW

**Comment by JameelB** [comment]
@vchepeli @twnolan

File: utils/file.crudl.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+module.exports.create = function() {
+  // var mobileBrowser = browser.forkNewDriverInstance(false, false);
+  // mobileBrowser.sleep(1000);
+  //
+  browser.get("http://localhost:9002/?url=http://localhost:8001");
+  browser.sleep(3000);
+
+
+  // browser.executeScript('window.open()').then(function() {
```
Comment (vchepeli) [inline utils/file.crudl.js]: @JameelB I tried this one too and it works but it does not navigate to right page. I still see that user is logged and [Log Out] button is shown

File: tests/file.spec.js [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+
+  describe('FILES NAVIGATION', function() {
+    it('should be able to navigate to the files section', function() {
+      expect(filePage.locators.header.isPresent()).eventually.to.be.true;
+      expect(filePage.locators.header.getText()).eventually.to.equal('Files');
+      expect(filePage.locators.emptyContent.isPresent()).eventually.to.be.true;
+    });
+  });
+
+  //Create - Mobile only
```
Comment (vchepeli) [inline tests/file.spec.js]: remove this comments

File: tests/file.spec.js [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+  describe('CREATE', function() {
```
Comment (vchepeli) [inline tests/file.spec.js]: make describe.skip and put //TODO comment there

File: tests/file.spec.js [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+    xit('should be able to click on the create button', function() {
+    });
+
+    xit('should be able to create a new file', function() {
```
Comment (vchepeli) [inline tests/file.spec.js]: if you make describe.skip you dont need to xit every test

**Comment by vchepeli** [comment]
LGTM

**Pull request merged**